### ISSUE TITLE
Create template for fortnightly weekly meet-up sessions

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutisticaCitizenScience#434]
## Summary

It could be helpful to have an open, standard template for fortnightly meet-up sessions. This is to:
 
* make the sessions consistent so people know what to expect
* make the sessions more efficient to organise going forwards
* make it easier for others to be onboarded to the project
* make it easier for people running their own projects to learn from this one

## Suggestions for tasks that can fix this issue

* [ ] Create a template on GitHub as an md file and put it somewhere sensible 
* [ ] Get it reviewed and merged
* [ ] Try it out 

## Who can help

**Beginners**

Leave a comment under this issue for suggestions for what should be included in the template or feedback from a session you attended or how the sessions are promoted and run. All responses appreciated, especially from autistic people! 

**Intermediate**

Review the PR of the template.

**Comment by GeorgiaHCA** [comment]
I've made a PR and added a lot of reviewers to the PR who I think would be best able to help - but if you're too busy don't worry! Lots and lots of thanks to @malvikasharan, @BrainonSilicon and @KirstieJane for your help already!
